spec/cc/yaml/nodes/engine_list_spec.rb

```diff
@@ -1,13 +1,39 @@
-require 'spec_helper'
+require "spec_helper"
 
 describe CC::Yaml::Nodes::EngineList do
-  specify 'with engines' do
+  specify "with engines" do
     config = CC::Yaml.parse! <<-YAML
-engines:
-  foo:
-    enabled: true
-    YAML
+      engines:
+        foo:
+          enabled: true
+          YAML
     config.engines.size.must_equal 1
     config.engines.keys.must_equal ["foo"]
   end
+
+  specify "with languages already present, it throws a warning" do
+    config = CC::Yaml.parse! <<-YAML
+      languages:
+      - "Ruby"
+      - "JavaScript"
+      engines:
+        rubocop:
+          enabled: true
+          YAML
```

Comment:
These terminating lines should be outdented to line up with the declaration. In this case, it should line up with `config =`